draft-ietf-httpbis-http2bis.xml

```diff
@@ -619,18 +619,40 @@
           which, if blocked by the transmission of a large frame, could affect performance.
         </t>
       </section>
-      <section anchor="HeaderBlock">
-        <name>Field Section Compression and Decompression</name>
+      <section anchor="FieldBlock">
+        <name>Header Compression and Decompression</name>
         <t>
-          A field in HTTP is a name with one or more associated values.
-          Field lines carry each value and are used within HTTP/2 request and response messages as well as in server push operations
-          (see <xref target="PushResources"/>).
+          The term "header compression", or its inverse of "header decompression", is used to refer
+          to the process of compressing, or decompressing, a set of field lines to form a field
```

Comment:
'or decompressing' reads weirdly to me, since that doesn't apply to the action described: field lines -> field block.  Might be better to just say compression transforms field lines to field blocks and decompression transforms field blocks into field lines.